CONVENTIONS.md

```diff
@@ -116,12 +116,72 @@ Name screen coordinates `x` and `y`, rather than `left` and `top`.
 
 ## Javadocs
 
-Write sentences for class, method and fields javadocs, starting with an uppercase and ending with a period. Start method docs with verbs, like `Gets` or `Called`. Use HTML tags such as `<p>` if the docs have several paragraphs.
+Write sentences for class, method and fields javadocs, starting with an uppercase and ending with a period. Start method docs with verbs, like `Gets` or `Called`. Use HTML tags such as `<p>` if the docs have several paragraphs, as line wraps are converted to spaces in the generated documentation. Feel free to start a new line whenever you feel the current line is too long.
 
-Write quick descriptions for parameter javadocs as well as `@return` tags, with no uppercase or period. Add parameter docs to the parameter itself instead of using the `@param` tag.
+Write quick descriptions for parameter javadocs as well as `@return` tags, with no uppercase or period. Add parameter docs to the parameter itself instead of adding `@param` tags to its owner method, except for type parameters (like `<T>` in `public class Lazy<T>`) whose descriptions can only go to `@param` tags of the classes or methods that declare them.
+
+Javadoc will take the first sentence, ended by the first `.`, as a brief description of the member you are documenting; you can use `{@summary }` inline tag to enclose another sentence to serve as the description, such as when you have `.` within sentences like `{@summary Returns the client's player, i.e. the controlled player.}`.
+
+### Tooling
+
+Fabric's official hosted javadocs are generated with [Java 11's standard doclet](https://docs.oracle.com/en/java/javase/11/docs/specs/doc-comment-spec.html) and can use any feature it supports, though you can personally build the documentation with newer Java. See ["checking javadoc"](#checking-javadoc) for how to build the documentation locally.
+
+### Custom tags
+
+A few additional block tags are supported:
+ - `@apiNote`: API Notes. A few comments for users of the documented member.
+ - `@implSpec`: Implementation Specification. Tells how this method should be implemented.
+ - `@implNote`: Implementation Notes. A few comments on the implementation.
+
+Feel free to use these tags and write under these sections.
+
+### Linking to other members
 
 Use `@link`, `@linkplain` and `@see` tags to refer to other parts of the code.
 
+When linking to another class, you can use the simple name of a class if it's from `java.lang` package or from the same package, or it appears in the signature of the class or its members (methods or fields).
+
```

Comment:
I'd prefer to link to https://docs.oracle.com/en/java/javase/15/docs/api/java.base/java/lang/Class.html#getSimpleName()

Reply:
```suggestion

If it does not fulfill one of these scenarios, use the full name, such as `com.google.common.collect.Lists` rather than simply `Lists`.

Use Yarn mappings when referencing Minecraft members, such as `net.minecraft.server.world.ThreadedAnvilChunkStorage` rather than `net.minecraft.class_3898`.

```
GitHub has a merge conflict